cashweb-payload/proto/payload.proto

```diff
@@ -47,3 +47,9 @@ message BurnTx {
     // the signed payload.
     uint32 burn_idx = 2;
 }
+
+// Set of auth wrappers for returning multiple items to the client as needed.
+message SignedPayloadSet {
+    // Set of auth wrappers that can be used in certain get responses
+    repeated SignedPayload items = 1;
```

Comment:
Should likely be a list of hashes rather than actual payloads so that syncing doesn't have to download irrelevant payloads.